lib/loader.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
   ScoreBoard: require('./scoreboard'),
   BossBar: require('./bossbar'),
   Particle: require('./particle'),
-  supportedVersions,
+  supportedVersions: testedVersions,
```

Comment:
I think that is not right

we support many more versions than the tested versions

let's expose oldestSupportedVersion and oldestSupportedVersion and stop exposing supportedVersions

it's slightly breaking but I think it should be ok